web/src/Components/CommentsZone/CommentsZone.js

```diff
@@ -0,0 +1,106 @@
+import React, { Component } from 'react';
```

Comment:
The comments order is not always the same! You can [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort](sort) them by creation date for example.

Reply:
Also, adding the creation date would be nice! (like pretty small on the top right corner, in color `#CCC`, and with a tooltip showing the last update date on hover :p)